Source/Plugins/CrossPlatformPluginGenerator/Assets/Microsoft.SpatialAudio.Spatializer.Unity/Plugins/x86/HrtfDsp.dll.meta

```diff
@@ -18,10 +18,10 @@ PluginImporter:
       settings:
         Exclude Android: 1
         Exclude Editor: 1
-        Exclude Linux64: 1
-        Exclude OSXUniversal: 1
-        Exclude Win: 1
-        Exclude Win64: 1
+        Exclude Linux64: 0
+        Exclude OSXUniversal: 0
```

Comment:
Shouldn't these be excluded for x86 binaries?

Reply:
I edited these files inside Unity editor. Don't think if these need to be explicitly excluded, there's no editor UI to exclude them?
![image](https://user-images.githubusercontent.com/13240863/160950926-2a4aa932-e429-4c2f-9b63-68af4f7e93f8.png)